osu.Game/Overlays/MusicController.cs

```diff
@@ -135,13 +137,26 @@ public bool TogglePause()
             return true;
         }
 
+        private PreviousButtonAction? prevAction;
```

Comment:
This should not be a field. The point of introducing this was to return an enum instead of a bool in `PrevTrack`.